docs/index.html

```diff
@@ -75,12 +75,13 @@ <h2 class="mb-5">Awareness Information</h2>
                     src="{{ '/assets/images/animation/annotations.png' | relative_url }}" />
             </div>
             <div class="col">
-                <p class="lead">As a user in a session, you the see:
+                <p class="lead">As a user in a session, you see:
                     <ul>
                         <li>the recent contribution of all team members<br /> <img class="img-fluid"
                                 src="{{ '/assets/images/contribution_annotation.png' | relative_url }}" /></li>
                         <li>the current selection of all team members<br /> <img class="img-fluid"
                                 src="{{ '/assets/images/selection_annotation.png' | relative_url }}" /></li>
+                        <li><a href="{{ '/documentation/awareness-information.html' | relative_url }}"><button type="button" class="btn btn-getting-started btn-sm">and more</button></a></li>
```

Comment:
As suggestion: Move button out of list.

```suggestion
                <p class="lead">As a user in a session, you see
                    <ul>
                        <li>the recent contribution of all team members,<br /> <img class="img-fluid"
                                src="{{ '/assets/images/contribution_annotation.png' | relative_url }}" /></li>
                        <li>the current selection of all team members,<br /> <img class="img-fluid"
                                src="{{ '/assets/images/selection_annotation.png' | relative_url }}" /></li>
                        <li>and more.</li>
                        <br />
                        <li style="list-style-type:none;"><a href="{{ '/documentation/awareness-information.html' | relative_url }}"><button type="button" class="btn btn-getting-started btn-sm">Awareness Information Overview</button></a></li>
```

Reply:
This was my first version, but I thought the button would look very separated.

Reply:
I think this is very subjective, but I think it looks to cramped with the current handling. Especially the in-line usage of a button as a link (with a bullet point before it) seems weird to me.

This way end the list and then offers the button leading to the awareness information page (labeled as such) for additional information.

You could also still play around with the positioning (i.e. don't have it be part of the list, etc.). I liked it this way as it gives the button kind of a centered look to match the "Getting Started" button at the top of the page.